lib/license_finder/package_managers/rebar_package.rb

```diff
@@ -0,0 +1,15 @@
+module LicenseFinder
+  class RebarPackage < Package
+    def initialize(name, version, install_path, dep, options={})
+      super(
+        name,
+        version,
+        options.merge(
+          description: name,
```

Comment:
It's OK to leave description blank if you can't fetch any more descriptive text from rebar.